appshell/client_handler.cpp

```diff
@@ -280,65 +278,6 @@ void ClientHandler::OnLoadingStateChange(CefRefPtr<CefBrowser> browser,
   SetNavState(canGoBack, canGoForward);
 }
 
-bool ClientHandler::OnConsoleMessage(CefRefPtr<CefBrowser> browser,
```

Comment:
Some changes here are not limited to Linux. I would recommend Linux only changes for this PR.